### PR TITLE
Removing mobterm event

### PR DESCRIPTION
**Pull request by yonetmen**

### CODE REVIEW

File: service/src/main/java/eu/europa/ec/fisheries/uvms/asset/domain/entity/CustomCode.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 @Entity
 @Table(name = "customcode"/*, indexes = { @Index(columnList = "id", name = "customcodes00", unique = true),}*/)
 @NamedQueries({
-        @NamedQuery(name = CUSTOMCODES_GETALLFOR, query = "SELECT m FROM CustomCode m where  upper(m.primaryKey.constant) =: constant"),
```
Comment (peerik) [inline service/src/main/java/eu/europa/ec/fisheries/uvms/asset/domain/entity/CustomCode.java:15]: Why is this removed?

Comment (yonetmen) [reply service/src/main/java/eu/europa/ec/fisheries/uvms/asset/domain/entity/CustomCode.java:15]: That is the one I discussed with Jonas and Thomas. That latest chance caused a test to fail. When I used the earlier version, test passed. I was going to ask you about it but apparently, I forgot about it.

**Comment by yonetmen** [comment]
test me

File: service/src/main/java/eu/europa/ec/fisheries/uvms/asset/domain/entity/CustomCode.java (hunk 1 above)
Comment (yonetmen) [reply service/src/main/java/eu/europa/ec/fisheries/uvms/asset/domain/entity/CustomCode.java:15]: I fixed the query and working as expected now.

**Review by ebrygg — APPROVED** [review]
ok!
